repo/packages/H/hdfs/0/marathon.json

```diff
@@ -4,12 +4,12 @@
   "mem": 512,
   "disk": 100,
   "instances": 1,
-  "cmd": "{{#hdfs.custom-config}}echo {{hdfs.custom-config}} | base64 --decode > mesos-site.xml && {{/hdfs.custom-config}}cd hdfs-mesos* && if [ -e '../mesos-site.xml' ]; then cp -f ../mesos-site.xml ./etc/hadoop/; fi && ./bin/hdfs-mesos",
+  "cmd": "export LD_LIBRARY_PATH=/opt/mesosphere/lib && export JAVA_HOME=$MESOS_DIRECTORY/jre1.7.0_76 && env && {{#hdfs.custom-config}}echo {{hdfs.custom-config}} | base64 --decode > mesos-site.xml && {{/hdfs.custom-config}}cd hdfs-mesos-* && if [ -e '../mesos-site.xml' ]; then cp -f ../mesos-site.xml ./etc/hadoop/; fi && ./bin/hdfs-mesos",
   "uris": {{{hdfs.scheduler-uris}}},
```

Comment:
I'm moving the setting of LD_LIBRARY_PATH to the "env" section.

> Just a question do we need && env here?
It's not strictly necessary, but nice to have for debugging purposes, so one can see what environment the scheduler sees.